cmd/tk/export.go

```diff
@@ -75,7 +75,7 @@ func exportCmd() *cli.Command {
 			}
 
 			// validate environment
-			if _, err := tanka.Peek(path, opts.ParallelOpts.JsonnetOpts); err != nil {
+			if _, err := tanka.Peek(path, opts.JsonnetOpts); err != nil {
 				return err
 			}
```

Comment:
Nit: `tanka.ListEnvs` only returns validated environments, so validating here again seems unnecessary.

Reply:
The `ListEnvs` is nested in the `if *recursive { }` block and has no affinity with this `Peek`.

Reply:
That's true, sorry for that